### PR TITLE
Allow copying text in block (non input)

### PR DESCRIPTION
**Pull request by ellatrix**
## Description
This PR fixes copying text form a block that is not inside an input field.

## How Has This Been Tested?
Create an image block. Select some text from the placeholder and copy it. Copied text should not be the whole block.

### CODE REVIEW

**Comment by iseulde** [comment]
@melchoyce This should fix your issue.

**Review by jorgefilipecosta — APPROVED** [review]
This tests well for me, and the code changes look good. Nice fix 👍

**Comment by iseulde** [comment]
Thanks!

File: editor/utils/dom.js [hunk 1]
```diff
@@ -324,3 +324,20 @@ export function isInputField( { nodeName, contentEditable } ) {
 		contentEditable === 'true'
 	);
 }
+
+/**
+ * Check wether the current document has a selection.
```
Comment (aduth) [inline editor/utils/dom.js]: Typo: "wether" -> "whether"

**Pull request merged**